src/models.ts

```diff
@@ -0,0 +1,49 @@
+export interface FileModel {
+  name?: '';
+  size?: '';
+  type?: '';
+  lastModified?: '';
+}
```

Comment:
Q: Is it useful to have these properties typed to be exclusively the empty string literal?